cmd/chains.go

```diff
@@ -93,20 +96,29 @@ $ %s ch s ibc-0 --yaml`, appName, appName, appName, appName)),
 			case yml && jsn:
 				return fmt.Errorf("can't pass both --json and --yaml, must pick one")
 			case yml:
-				out, err := yaml.Marshal(c)
+				pcfgw := &ProviderConfigWrapper{
+					Type:  c.ChainProvider.Type(),
+					Value: c.ChainProvider.ProviderConfig(),
+				}
+				out, err := yaml.Marshal(pcfgw)
 				if err != nil {
 					return err
 				}
 				fmt.Println(string(out))
 				return nil
 			case jsn:
-				out, err := json.Marshal(c)
+				pcfgw := &ProviderConfigWrapper{
+					Type:  c.ChainProvider.Type(),
+					Value: c.ChainProvider.ProviderConfig(),
+				}
+				out, err := json.Marshal(pcfgw)
 				if err != nil {
 					return err
 				}
 				fmt.Println(string(out))
 				return nil
 			default:
+				// TODO still need to tweak this for new config
```

Comment:
We can actually just pull this default case out and go only with JSON or YAML if you want too. IDK if the text representation is useful.

Reply:
I made yaml the default case and left in a flag for json